src/database/database.module.ts

```diff
@@ -0,0 +1,29 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import {
+  databaseName,
+  dialect,
+  hostDB,
+  passwordDB,
+  portDB,
+  typeDB,
```

Comment:
Please, use NestJS config module for infrastructure constants.
https://docs.nestjs.com/techniques/configuration#using-the-configservice